src/tests/integration-test

```diff
@@ -96,7 +96,7 @@ class UDisksTestCase(unittest.TestCase):
     manager = None
 
     @classmethod
-    def init(klass, logfile=None):
+    def init(cls, logfile=None):
```

Comment:
I don't understand the commit message. Do you mean they should be methods of instances? Cause they are *class methods* now.